CHANGELOG.md

```diff
@@ -1,5 +1,237 @@
 # Changelog
 
+## 0.36.0
+
+### Breaking Changes
+
+* (rest) [\#4837](https://github.com/cosmos/cosmos-sdk/pull/4837) Remove /version and /node_version
+  endpoints in favor of refactoring /node_info to also include application version info.
+* All REST responses now wrap the original resource/result. The response
+  will contain two fields: height and result.
+* [\#3565](https://github.com/cosmos/cosmos-sdk/issues/3565) Updates to the governance module:
+  * Rename JSON field from `proposal_content` to `content`
+  * Rename JSON field from `proposal_id` to `id`
+  * Disable `ProposalTypeSoftwareUpgrade` temporarily
+* [\#3775](https://github.com/cosmos/cosmos-sdk/issues/3775) unify sender transaction tag for ease of querying
+* [\#4255](https://github.com/cosmos/cosmos-sdk/issues/4255) Add supply module that passively tracks the supplies of a chain
+  - Renamed `x/distribution` `ModuleName`
+  - Genesis JSON and CLI now use `distribution` instead of `distr`
+  - Introduce `ModuleAccount` type, which tracks the flow of coins held within a module
+  - Replaced `FeeCollectorKeeper` for a `ModuleAccount`
```

Comment:
I would link to the docs of the supply module, this is fairly vague for a breaking change, but this changelog is so big people may avoid it